src/PlusDataCollection/Andor/vtkPlusAndorVideoSource.cxx

```diff
@@ -591,6 +591,11 @@ void vtkPlusAndorVideoSource::AddFrameToDataSource(DataSourceArray& ds)
       LOG_INFO("Success adding item to AndorCamera video source " << ds[i]->GetSourceId());
     }
   }
+  unsigned status = checkStatus(FreeInternalMemory(), "FreeInternalMemory");
```

Comment:
Why do this? It will need to be allocated for the next frame, probably.

Reply:
It was one of the things we tried when first debugging the issue and was just left in, but I can take it out. It shouldn't be related to the main issue at hand.

>Filtering for a device can be disabled by setting it to 1 or 0, I think.

Which setting are you thinking of? Is it a member of vtkPlusDevice?

Reply:
AveragedItemsForFiltering, in e.g. http://perk-software.cs.queensu.ca/plus/doc/nightly/dev/DeviceCapistrano.html

Reply:
Hmm I see so when this is set <2 the unfiltered timestamp is just used as the "filtered" timestamp instead. Do you think this should just be a change in our device config? I think that would mean that anytime that `AveragedItemsForFiltering` is set to anything other than 0 or 1 in the config, then acquisition will still fail (if somebody else uses the device for example), so shouldn't the forced unfiltered timestamp be done here as well?

Reply:
Somebody else could be using higher frame rates? But since the device is meant for longer exposures, doing it with this patch makes more sense.